tests/features/package-explorer.feature

```diff
@@ -0,0 +1,61 @@
+Feature: Package Explorer - View details about Package
+  User can navigate to and view details of what TPA knows about individual Package
```

Comment:
Shall we use the Goal as Feature like "As a developer I want to view all of the Package metadata, vulnerabilities impacting, products containing."

Reply:
Not sure what You mean here - to change/clarify the description?

As I do not see 'Goal' keyword https://cucumber.io/docs/gherkin/reference/#keywords

Reply:
Ah, it is from the user stories doc.